commands/src/api/mod.rs

```diff
@@ -49,8 +50,8 @@ pub enum Body {
     StartDeployment(deployment::StartDeployment),
     /// Stop deployment
     StopDeployment(deployment::StopDeployment),
-    /// Discover cameras
-    Discover,
```

Comment:
The commit log doesn't explain removal of this.

Reply:
Yep, will add something like
> `Discover` was moved under `mod camera`

at rebase.